### ISSUE TITLE
can't compile with g++

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Neopallium/llvm-lua#1]
```
On Ubuntu 9.04, with gcc/g++ 4.3.3, 
the compilation fails with the following message :

$ make linux
cd llvm-lua && make linux
make[1]: Entering directory `/home/user/checkout/llvm-lua-0.5/llvm-lua'
make all MYCFLAGS=-DLUA_USE_LINUX MYLIBS="-rdynamic -Wl,-E -ldl "
make[2]: Entering directory `/home/user/checkout/llvm-lua-0.5/llvm-lua'
g++ -c -I../src -O3 -fomit-frame-pointer -pipe -Wall -DLUA_NODEBUG
-DUSE_BITCODE_FILE -DLUA_USE_LINUX  `llvm-config --cppflags` -o
load_vm_ops.o load_vm_ops.cpp
load_vm_ops.cpp: In function ‘llvm::ModuleProvider* load_vm_ops(bool)’:
load_vm_ops.cpp:74: error: ‘exit’ was not declared in this scope
load_vm_ops.cpp:84: error: ‘exit’ was not declared in this scope
load_vm_ops.cpp:91: error: ‘exit’ was not declared in this scope
make[2]: *** [load_vm_ops.o] Error 1


```

Original issue reported on code.google.com by `fper...@gmail.com` on 15 May 2009 at 5:28

**Comment by GoogleCodeExporter** [comment]
```
try adding:
#include <stdlib.h>

to the top of load_vm_ops.cpp

Are you using the 0.5 release?

```

Original comment by `rjakabo...@gmail.com` on 15 May 2009 at 6:02

**Comment by GoogleCodeExporter** [comment]
```

with #include <stdlib.h>, the compilation is fixed.

Yes, I use 0.5 release.

```

Original comment by `fper...@gmail.com` on 16 May 2009 at 11:28

**Comment by GoogleCodeExporter** [comment]
```
Fixed in new Release 1.0.
```

Original comment by `rjakabo...@gmail.com` on 31 May 2009 at 4:35
- Changed state: **Fixed**
